api/src/MemberMatch/Scorer.cs

```diff
@@ -8,145 +8,147 @@ namespace RaceResults.MemberMatch
 {
     public class Scorer
     {
-        static public double defaultPriorScore = -11.51291546;
-        static public double defaultProbabilityAppearsInLineFromReference = 0.6;
-        static private double defaultRare = 1e-5;
+        public static double DefaultPriorScore = -11.51291546;
+        public static double DefaultProbabilityAppearsInLineFromReference = 0.6;
+        private static double defaultRare = 1e-5;
 
         public Scorer(Dictionary<string, double> nameToProbability)
         {
-            _nameToProbability = nameToProbability;
+            this.nameToProbability = nameToProbability;
         }
 
-
         public Scorer(string nameToProbabilityFileName)
         {
-            _nameToProbability = new Dictionary<string, double>();
+            nameToProbability = new Dictionary<string, double>();
             int index = -1;
             foreach (string line in File.ReadLines(nameToProbabilityFileName))
             {
                 index++;
-                if (index==0)
+                if (index == 0)
                 {
                     continue;
                 }
+
                 var fields = line.Split('\t');
-                _nameToProbability[fields[0]] = double.Parse(fields[1]);
+                nameToProbability[fields[0]] = double.Parse(fields[1]);
             }
         }
 
-        private Dictionary<string, double> _nameToProbability;
-
-        static public double Delta(
-                                   double[] probabilityAppearsInLineByCoincidenceList,
-                                   bool[] isContainedList,
-                                   double[] probabilityAppearsInLineFromReferenceList = default)
+        private Dictionary<string, double> nameToProbability;
 
+        public static double Delta(
```

Comment:
nit: first param should be on this line

Reply:
or not indented

Reply:
That's what I thought, too, but StyleCop says:
"The parameters should begin on the line after the declaration, whenever the parameter span across multiple lines:"
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1116.md